### ISSUE TITLE
How to pass airflow task configuration to one custom component?

### ISSUE THREAD

**Issue by Natielle** [tensorflow/tfx#6438]
**System information**

- TFX Version (you are using): 0.22.0
- Environment in which you plan to use the feature (e.g., Local
  (Linux/MacOS/Windows), Interactive Notebook, Google Cloud, etc..): Linux/Mac
- Are you willing to contribute it (Yes/No): Yes

**Describe the feature and the current behavior/state.**
I would like to use the `trigger_rule` Airflow configuration in only one component. I know it is possible for all components from pipeline using the parameter `airflow_dag_config` (from this object `AirflowPipelineConfig`). 
Is it possible to use `trigger_rule` configuration only to one component? 
My component execution should be optional. This component is used to send updates to another system, if this optional component fails, it will not be able to interrupt the execution of the main tasks.

**Comment by singhniraj08** [comment]
@Natielle,

I don't think we have an option to add "trigger_rule" Airflow configuration in only one TFX component. Similar #498 feature request can help you use-case but it's still a work in progress. I would request you to close this issue and follow similar thread for updates on the feature implementation. Thank you!
